test/ttmlir/Conversion/TTIRToD2M/sign_op_verifier.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: not ttmlir-opt %s 2>&1 | FileCheck %s
+
```

Comment:
Oh, interesting, I have never done invalid (error) tests that way. How does it compare to using `-verify-diagnostics` and `// expected-error`?